brave/brave-core
# Cleanup cosmetic filter code

Remove unneeded reducers/filters/types/etc

Code was changed at some point (in the API) to directly use chrome storage APIs (not using redux / store)

NOTE: no ticket created, since this is a dev concern (doesn't need manual QA or to be included in release notes); but I can create one if it's useful

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Launch browser and visit brave.com
2. Right click the orange `Download Brave` button, pick `Brave` => `Block element via selector`
3. Input box shows; click OK
4. Element should disappear
5. Reload site to verify it's still gone
6. Right click anywhere on the page and pick `Brave` => `Clear CSS rules for this site`
7. Reload page
8. Verify button shows again
9. Repeats steps 2-5
10. Right click anywhere on the page and pick `Brave` => `Clear CSS rules for all sites`
11. Reload page
12. Verify button shows again

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
